### ISSUE TITLE
Vanilla rails and railcraft

### ISSUE THREAD

**Issue by sigurdtheone** [John-Smith-Modded/JSTR-Modded-1.10.x#4]
There are currently some issues with the compatibility of the vanilla mine cart blockstates and railcraft causing us to have to delete these blockstates from vanilla in order to display railcraft textures properly

**Comment by Shoeboxam** [comment]
Railcraft references the default model file, which is modified in JS.
`assets\railcraft\blockstates\tracks\outfitted\kit\`

I encountered the same issue with Soartex and Invictus. I fixed it by removing all modifications to default model files, and redirecting model edits to alternate locations. Sample blockstate:
https://github.com/InvictusGraphics/Invictus_Textures/blob/master/assets/minecraft/blockstates/rail.json

Models folder, where all models are tucked into folders to prevent them from overwriting vanilla models:
https://github.com/InvictusGraphics/Invictus_Textures/tree/master/assets/minecraft/models/block

The best solution is to restructure the vanilla pack so that it doesn't make modifications to vanilla model files. The next best workaround is to just copy the default model files into the mod patch, so that they revert to original behavior.
